src/main/lrsql/system/webserver.clj

```diff
@@ -65,7 +65,9 @@
         ;; default ports
         allowed-list
         (or allowed-origins
-            (cond-> [(format "http://%s:%s" http-host http-port)
+            (cond-> [(format "http://localhost:%s" http-port)
+                     (format "https://localhost:%s" ssl-port)
```

Comment:
And by the same principle, there's no need to add `http:0.0.0.0:%s" as one of the default allowed origins.

Reply:
Since that is the default bind host, not needed yeah